x-pack/plugins/security_solution/cypress/integration/detection_alerts/alerts_details.spec.ts

```diff
@@ -23,7 +23,7 @@ import { unmappedRule } from '../../objects/rule';
 import { DETECTIONS_URL } from '../../urls/navigation';
 
 describe('Alert details with unmapped fields', () => {
-  before(() => {
+  beforeEach(() => {
```

Comment:
Is this change necessary? I agree it's cleaner to do this on each test run, but it also adds additional time to running this test.

Reply:
Yes, this test was discussed with @MadameSheema.
This test is flaky, she suggested using `beforeEach` instead of `before`, because this way the test is retried if it fails. Although it takes a little bit more time to complete.